.github/ci-scripts/bash/functions-for-build

```diff
@@ -0,0 +1,166 @@
+#!/bin/bash
+
+source ./ci-scripts/bash/functions-for-build-camel-case
+
+create_pip_conf(){
+  echo "[global]" > pip.conf
+  echo "index =  ${NEXUS_PROTOCOL}://${NEXUS_USERNAME}:${NEXUS_PASSWORD}@${NEXUS_SERVER_ENDPOINT}/repository/${NEXUS_PULL_REPOS_PY}/simple" >> pip.conf
+  echo "index-url = ${NEXUS_PROTOCOL}://${NEXUS_USERNAME}:${NEXUS_PASSWORD}@${NEXUS_SERVER_ENDPOINT}/repository/${NEXUS_PULL_REPOS_PY}/simple" >> pip.conf
+  echo "extra-index-url = https://pypi.org/simple" >> pip.conf
+  sudo cp pip.conf /etc/pip.conf
+  cat /etc/pip.conf
+}
+
+docker_build() {
+  set -e
+  IMAGE="${1}"
+  FILE="${4}"
+  if [[ "${FILE}" == "null" || -z "${FILE}" ]]; then
+    FILE="./Dockerfile"
+  fi
+  required_inputs="1"
+  validate_inputs "${required_inputs}" "$@" 
+  docker build -f ${FILE} -t ${IMAGE} \
+              --build-arg GITHUB_USER=${GITHUB_USER} \
+              --build-arg GITHUB_TOKEN=${GITHUB_TOKEN} \
+              --build-arg NEXUS_PROTOCOL=${NEXUS_PROTOCOL} \
+              --build-arg NEXUS_SERVER_ENDPOINT=${NEXUS_SERVER_ENDPOINT} \
+              --build-arg NEXUS_USERNAME=${NEXUS_USERNAME} \
+              --build-arg NEXUS_PASSWORD=${NEXUS_PASSWORD} .
+  set +e
+}
+
+docker_build_and_push() {
+  set -e
+  IMAGE="${1}"
+  VERSION="${3}"
+  TAG=${LATEST_TAG}
+  if [[ "${VERSION}" != "null" ]]; then
+    TAG=${VERSION}
+  fi
+  docker_build "$@"
+  docker tag ${IMAGE}:latest ${DOCKERHUB_ORG}/${IMAGE}:${TAG}
+  docker push ${DOCKERHUB_ORG}/${IMAGE}:${TAG}
+  set +e
+}
+
+mvn_deploy() {
+  ARTIFACT_ID="${1}"
+  EXTENSION="${2}"
+  FILE="${3}"
+  GROUP_ID="${4}"
+  VERSION="${5}"
+  required_inputs="1 2 3 4 5"
+  validate_inputs "${required_inputs}" "$@" 
+  mvn deploy:deploy-file -Dnexus_url=${NEXUS_PROTOCOL}://${NEXUS_SERVER_ENDPOINT}/repository/tessell-m2-component \
+      -Dnexus_username=${NEXUS_USERNAME} -Dnexus_password=${NEXUS_PASSWORD} \
+      -DgroupId=${GROUP_ID} -DartifactId=${ARTIFACT_ID} -Dversion=${VERSION} \
+      -DgeneratePom=true -Dpackaging=${EXTENSION} \
+      -Durl=${NEXUS_PROTOCOL}://${NEXUS_SERVER_ENDPOINT}/repository/${NEXUS_PUSH_REPOS_M2} \
+      -Dfile=${FILE} -DrepositoryId=nexus
+}
+
+qa_build_and_push(){
+  set -e
+  required_inputs="1 3"
+  validate_inputs "${required_inputs}" "$@" 
+  create_pip_conf
```

Comment:
We are now not creating this file. Please run the build without creating /etc/pip.conf and see if it works. If it fails we need to add `--extra-index-url` to the `pip` command directly.